app/views/assets/samples_partials/_asset_samples.erb

```diff
@@ -8,10 +8,11 @@
     </h3>
     </a>
   </header>
+  <div id="asset-samples-table-collapser" class="collapse in show">
```

Comment:
I think the `card-body` div is not indented correctly?

Reply:
I moved the collapser div up to include the comment about the tags in the collapsed section as well. Otherwise, you collapse the section but the comment related to the section stays visible.

"If the tags below are not correct, you may wish to <%= link_to 'perform tag substitution', new_receptacle_tag_substitution_path(@asset.receptacle) %>."